server/routes/record-attendance/handlers/attendanceList.test.ts

```diff
@@ -38,69 +37,70 @@ describe('Route Handlers - Attendance List', () => {
       },
       body: {},
     } as unknown as Request
+
+    when(activitiesService.getScheduledActivity)
+      .calledWith(1, res.locals.user)
+      .mockResolvedValue({
+        id: 1,
+        date: format(new Date(), 'yyyy-MM-dd'),
+        startTime: '10:00',
+        endTime: '11:00',
+        activitySchedule: {
+          activity: { summary: 'Maths level 1' },
+          internalLocation: { description: 'Houseblock 1' },
+        },
+        attendances: [
+          { prisonerNumber: 'ABC123', status: 'WAITING' },
+          { prisonerNumber: 'ABC321', status: 'COMPLETED', attendanceReason: { code: 'ATTENDED' } },
+          { prisonerNumber: 'ZXY123', status: 'COMPLETED', attendanceReason: { code: 'SICK' } },
+        ],
+      } as ScheduledActivity)
+
+    when(activitiesService.getScheduledEventsForPrisoners)
+      .calledWith(expect.any(Date), ['ABC123', 'ABC321', 'ZXY123'], res.locals.user)
+      .mockResolvedValue({
+        activities: [
+          { eventId: 1, eventDesc: 'Maths', startTime: '10:00', endTime: '11:00', prisonerNumber: 'ABC123' },
+        ],
+        appointments: [
+          {
+            eventId: 2,
+            eventDesc: 'Appointment with the guv',
+            startTime: '15:00',
+            endTime: '16:00',
+            prisonerNumber: 'ABC123',
+          },
+        ],
+        courtHearings: [
+          { eventId: 3, eventDesc: 'Video link', startTime: '09:00', endTime: '10:00', prisonerNumber: 'ABC123' },
+          { eventId: 4, eventDesc: 'Video link', startTime: '10:30', endTime: '11:00', prisonerNumber: 'ABC321' },
+        ],
```

Comment:
Not important in the grand scheme - but for info, Video link bookings do not actually appear in COURT HEARINGS, as far as I know. COURT HEARINGS holds the real, in-person court hearings and VLBs actually show up as types of appointments in NOMIS.